package.json

```diff
@@ -8,9 +8,9 @@
   "repository": "Polymer/pwa-starter-kit",
   "scripts": {
     "start": "polymer serve",
-    "build": "npm run build:prpl-server && npm run build:static",
-    "build:prpl-server": "polymer build --auto-base-path && rm -rf server/build && mv build server/ && renamer --find node_modules --replace node_assets 'server/build/**' && replace node_modules node_assets server/build -r -s",
-    "build:static": "polymer build",
+    "build": "gulp build",
+    "build:prpl-server": "gulp clean:prpl-server && gulp build:prpl-server",
+    "build:static": "gulp clean:static && gulp build:static",
```

Comment:
why does `build:static` need to change? it doesn't have to do any work, so it doesn't need to be built by `gulp`

Reply:
I guess technically `build:static` is still functionally equivalent to `polymer build`